.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish package to GitHub Packages
+on:
+  push:
+    branches: [ main ]
```

Comment:
Could we limit this to just 'releases' Github style or tags with a version number, and avoid erroring out for every other commit?

Reply:
We could use tags, but it also means updating the package.json to match the tag value which seems like an opportunity to only do one or the other and get unexpected results.

I'll check out the github action releases events and see if that's less error prone.